### PR TITLE
Words replacements

### PR DESCRIPTION
**Pull request by themr0c**
- plugin -> plug-in
- Theia -> Che-Theia
- operator -> Operator

### CODE REVIEW

File: src/main/pages/che-7/contributor-guide/proc_developing-che-theia-plug-ins-using-che.adoc [hunk 1]
```diff
@@ -74,7 +74,7 @@ image::extensibility/running-plug-in-hosted-mode.gif[link="{imagesdir}/extensibi
 .. Using the command palette, stop your *Hosted Plug-in* instance (if any) by running the `Hosted Plugin: Stop Instance` command.
 .. Start the *Hosted Plug-in* instance again in the debugging mode by running the `Hosted Plugin: Debug Instance` command.
 .. Switch to the *Hosted Instance* tab and perform the required actions to reach the code with the breakpoint.
-.. Go back to the *Main Theia* window, and use the *Debug* panel as you need.
+.. Go back to the *Main Che-Theia* window, and use the *Debug* panel as you need.
```
Comment (rkratky) [inline src/main/pages/che-7/contributor-guide/proc_developing-che-theia-plug-ins-using-che.adoc:77]: This looks like a GUI element -- we need to observe what's the actual name of the window.

File: src/main/pages/che-7/end-user-guide/assembly_using-alternative-ides-in-che.adoc [hunk 2]
```diff
@@ -28,14 +28,14 @@ Extending Eclipse Che developer workspaces using different IDEs (integrated deve
...
 Eclipse Che provides a default web IDE to be used with the developer workspaces. This IDE is completely decoupled. You can bring your own custom IDE for Eclipse Che:
 
-* *Built from Eclipse Theia*, which is a framework to build web IDEs. Example: link:https://www.youtube.com/watch?v=B6aCqywKpyY[Sirius on the web].
+* *Built from Eclipse Che-Theia*, which is a framework to build web IDEs. Example: link:https://www.youtube.com/watch?v=B6aCqywKpyY[Sirius on the web].
 // TODO: Learn more link:[LINK].
 
 * *Completely different web IDEs*, such as Jupyter, Eclipse Dirigible, or others. Example: link:https://www.youtube.com/watch?v=VooNzKxRFgw[Jupyter in Eclipse Che workspaces].
 
-.Bringing custom IDE built from Eclipse Theia
+.Bringing custom IDE built from Eclipse Che-Theia
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/assembly_using-alternative-ides-in-che.adoc]: Eclipse Theia != Che-Theia. Sorry, I should've been more specific.

File: src/main/pages/che-7/end-user-guide/assembly_using-alternative-ides-in-che.adoc [hunk 3]
```diff
@@ -28,14 +28,14 @@ Extending Eclipse Che developer workspaces using different IDEs (integrated deve
...
 
-* Creating your own custom IDE based on Eclipse Theia
+* Creating your own custom IDE based on Eclipse Che-Theia
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/assembly_using-alternative-ides-in-che.adoc]: Eclipse Theia != Che-Theia.

File: src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc [hunk 4]
```diff
@@ -1,9 +1,9 @@
 [id="che-theia-plug-in-concept-in-detail_{context}"]
 = Che-Theia plug-in concept in detail
 
-Eclipse Che provides a default web IDE for workspaces: Che-Theia. It is based on Eclipse Theia. It is a slightly different version than the plain Eclipse Theia one because there are functionalities that have been added based on the nature of the Eclipse Che workspaces. This version of Eclipse Theia for Che is called *Che-Theia*.
+Eclipse Che provides a default web IDE for workspaces: Che-Theia. It is based on Eclipse Che-Theia. It is a slightly different version than the plain Eclipse Che-Theia one because there are functionalities that have been added based on the nature of the Eclipse Che workspaces. This version of Eclipse Che-Theia for Che is called *Che-Theia*.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc]: Eclipse Theia != Che-Theia.

File: src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
 
-You can extend the IDE provided with Eclipse Che by building a *Che-Theia plug-in*. Che-Theia plug-ins are compatible with any other Eclipse Theia-based IDE.
+You can extend the IDE provided with Eclipse Che by building a *Che-Theia plug-in*. Che-Theia plug-ins are compatible with any other Eclipse Che-Theia-based IDE.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc]: Eclipse Theia != Che-Theia.

File: src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc [hunk 6]
```diff
@@ -69,13 +69,13 @@ Che-Theia plug-ins have the following capabilities:
 
 
 [id="vs-code-extensions-and-eclipse-theia-plug-ins_{context}"]
-== VS Code extensions and Eclipse Theia plug-ins
+== VS Code extensions and Eclipse Che-Theia plug-ins
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc]: Eclipse Theia != Che-Theia.

File: src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc [hunk 7]
```diff
@@ -69,13 +69,13 @@ Che-Theia plug-ins have the following capabilities:
...
 
-A Che-Theia plug-in can be based on a VS Code extension or an Eclipse Theia plug-in.
+A Che-Theia plug-in can be based on a VS Code extension or an Eclipse Che-Theia plug-in.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc]: Eclipse Theia != Che-Theia.

File: src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc [hunk 8]
```diff
@@ -69,13 +69,13 @@ Che-Theia plug-ins have the following capabilities:
...
 
 A Visual Studio Code extension:: To repackage a VS Code extension as a Che-Theia plug-in with its own set of dependencies, package the dependencies into a container. This ensures that Eclipse Che users do not need to install the dependencies when using the extension. See link:{site-baseurl}che-7/using-a-visual-studio-code-extension-in-che[Using a Visual Studio Code extension in Che].
 
-An Eclipse Theia plug-in:: You can build a Che-Theia plug-in by implementing an Eclipse Theia plug-in and packaging it to Eclipse Che.
+An Eclipse Che-Theia plug-in:: You can build a Che-Theia plug-in by implementing an Eclipse Che-Theia plug-in and packaging it to Eclipse Che.
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/con_che-theia-plug-in-concept-in-detail.adoc]: Eclipse Theia != Che-Theia.

File: src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc [hunk 9]
```diff
@@ -57,8 +57,8 @@ version: 1.0.0
 type: Che Editor
 name: theia-ide
 id: org.eclipse.che.editor.theia
-title: Eclipse Theia for Eclipse Che
-description: Eclipse Theia
+title: Eclipse Che-Theia for Eclipse Che
+description: Eclipse Che-Theia
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc]: Eclipse Theia != Che-Theia.

File: src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc [hunk 10]
```diff
@@ -126,7 +126,7 @@ Describes plug-ins in a workspace by defining their `id`. It is allowed to have
      id: eclipse/che-machine-exec-plugin/0.0.1
 ----
 
-Both types above using id, which is slash-separated publisher, name and version of plugin from Che Plugin registry. +
+Both types above using id, which is slash-separated publisher, name and version of plug-in from Che Plugin registry. +
```
Comment (rkratky) [inline src/main/pages/che-7/end-user-guide/ref_devfile-reference.adoc]:
```suggestion
Both types above use an ID, which is slash-separated publisher, name and version of plug-in from the Che Plug-in registry. +
```

File: src/main/pages/che-7/overview/assembly_introduction-to-eclipse-che.adoc [hunk 11]
```diff
@@ -121,10 +121,10 @@ Eclipse Che is extensible in different ways:
 
 * *Plug-ins* to add capabilities to the IDE. Che-Theia plug-ins rely on APIs compatible with Visual Studio Code. Plug-ins are isolated and provide their own dependencies packaged in containers.
 * *Stacks* to create pre-configured Che workspaces with a dedicated set of tools.
-* *Alternative IDEs* to provide specialized tooling within Eclipse Che. Build your own, based on Eclipse Theia, or pick existing ones like Jupyter.
+* *Alternative IDEs* to provide specialized tooling within Eclipse Che. Build your own, based on Eclipse Che-Theia, or pick existing ones like Jupyter.
```
Comment (rkratky) [inline src/main/pages/che-7/overview/assembly_introduction-to-eclipse-che.adoc]: Eclipse Theia != Che-Theia.

**Review by rkratky — CHANGES_REQUESTED** [review]

File: src/main/pages/che-7/contributor-guide/proc_developing-che-theia-plug-ins-using-che.adoc (hunk 1 above)
Comment (themr0c) [reply src/main/pages/che-7/contributor-guide/proc_developing-che-theia-plug-ins-using-che.adoc:77]: AFAIK, this window has not a name. Not visible either in https://www.eclipse.org/che/docs/che-7/developing-che-theia-plug-ins/. So I believe this change is safe.

**Review by rkratky — APPROVED** [review]

**Pull request merged**